File: python/7 kyu/121.py
# https://www.codewars.com/kata/5738f5ea9545204cec000155/train/python

# Passed

def count_letters_and_digits(s):
    count = 0
    for char in s:
        if char.isalpha() or char.isdigit():
            count += 1

    return count

output = count_letters_and_digits('de?=?=tttes!!t')
print(output)

File: nodejs/7 kyu/121.js
// https://www.codewars.com/kata/54557d61126a00423b000a45/train/javascript

// Passed

function shorterReverseLonger(a, b) {
  let longerString = b.length > a.length ? b : a;
  let shorterString = longerString === a ? b : a;
  let result =
    shorterString + longerString.split("").reverse().join("") + shorterString;
  return result;
}

const output = shorterReverseLonger("first", "abcde");
console.log(output);